app/test_main.py

```diff
@@ -1,3 +1,61 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestPetAgeInHumanYears:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_list",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="given zero age"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="given age is under 15"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="given age is equal to 15"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="given age is under 24"
+            ),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="given age is equal to 24"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
+                id="given age is under cat and dog limitation"
```

Comment:
Id of this test doesn't seem very clear to me. Consider changing it.

Reply:
I agree with you. It's better to change it. Thank you!